web/packages/shared/components/MenuLogin/MenuLogin.tsx

```diff
@@ -127,6 +127,7 @@ const LoginItemList = ({
       <Input
         p="2"
         m="2"
+        name="notsearch_password"
```

Comment:
can we add explanation here? because i found it pretty strange too

Reply:
Done!